dali/python/nvidia/dali/ops.py

```diff
@@ -950,6 +950,17 @@ def _arithm_op(name, *inputs):
     # Call it immediately
     return op(*dev_inputs)
 
+
+# TODO(klecki): do we want to place it somewhere in a module?
+def min(left, right):
+    return _arithm_op("min", left, right)
+
+def max(left, right):
+    return _arithm_op("max", left, right)
+
+def clamp(value, lo, hi):
+    return _arithm_op("clamp", value, lo, hi)
```

Comment:
Maybe they should land in dali.fn? Both the usage mode and lowercase naming would play better with dali.fn.